packages/dubbo-serialization/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@apache/dubbo-serialization",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
We can still keep the 0.0.1 version because we have not released the npm module yet